scripts/error-codes/invertObject.js

```diff
@@ -3,19 +3,16 @@
  *
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
- *
- * @flow
  */
 'use strict';
 
-import type {ErrorMap} from './Types';
```

Comment:
This used a "type comment" before of the form:

```
/*::
[flow only code here]
*/
```
The `flow` prettier parser parses this as a pure AST w/o the comment and then when it's printed the comment is removed. Most of the scripts don't use types, I think we don't loose much by removing this here.